wcc-contentful/spec/wcc/contentful/store/postgres_store_spec.rb

```diff
@@ -26,6 +30,8 @@
   it_behaves_like 'contentful store', {
     nested_queries: true,
     include_param: true,
+    # TODO: - fix locale queries
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/watermarkchurch/wcc-contentful/blob/62b6395716d993b0c3c236623b7ccb14e8779f31/wcc-contentful/spec/wcc/contentful/store/postgres_store_spec.rb#L33"></span>TODO added in <a href="https://github.com/watermarkchurch/wcc-contentful/blob/62b6395716d993b0c3c236623b7ccb14e8779f31/wcc-contentful/spec/wcc/contentful/store/postgres_store_spec.rb">wcc-contentful/spec/wcc/contentful/store/postgres_store_spec.rb</a> - is there a card associated with that?</td>
    </tr>
  </tbody>
</table>